clang/labs/lab1/shishkina_valeriya/PrintClasses.cpp

```diff
@@ -0,0 +1,56 @@
+#include "clang/AST/ASTConsumer.h"
+#include "clang/AST/RecursiveASTVisitor.h"
+#include "clang/Frontend/CompilerInstance.h"
+#include "clang/Frontend/FrontendPluginRegistry.h"
+
+class PrintClassesVisitor : public clang::RecursiveASTVisitor<PrintClassesVisitor> {
```

Comment:
Please fix diff suggestions of clang-format test: https://github.com/NN-complr-tech/llvm-nnsu-2024/actions/runs/8310037939/job/22742071220?pr=18#step:4:14